doc/source/user_guide/tutorials/mesh/explore_mesh.rst

```diff
@@ -0,0 +1,283 @@
+
+    .. tab-item:: LSDYNA
+
+        .. jupyter-execute::
+
+            # Get the mesh nodes
+            nodes_2 = meshed_region_2.nodes
+
+            # Print the object type
+            print("Object type: ",type(nodes_2),'\n')
+
+            # Print the nodes
+            print("Nodes: ", nodes_2)
+
+    .. tab-item:: Fluent
+
+        .. jupyter-execute::
+
+            # Get the mesh nodes
+            nodes_3 = meshed_region_3.nodes
+
+            # Print the object type
+            print("Object type: ",type(nodes_3),'\n')
+
+            # Print the nodes
+            print("Nodes: ", nodes_3)
+
+    .. tab-item:: CFX
+
+        .. jupyter-execute::
+
+            # Get the mesh nodes
+            nodes_4 = meshed_region_4.nodes
+
+            # Print the object type
+            print("Object type: ",type(nodes_4),'\n')
+
+            # Print the nodes
+            print("Nodes: ", nodes_4)
+
+Explore the mesh metadata
+-------------------------
+
+You can access the mesh metadata by manipulating the |MeshedRegion| object properties.
+
+The mesh metadata information describes the mesh composition.
+
+You can access which metadata information is available for a given result file.
+
+.. tab-set::
+
+    .. tab-item:: MAPDL
+
+        .. jupyter-execute::
+
+            # Get the available properties
+            available_props_1 = meshed_region_1.available_property_fields
+
+            # Print the available properties
+            print("Available properties: ", available_props_1)
+
+    .. tab-item:: LSDYNA
+
+        .. jupyter-execute::
+
+            # Get the available properties
+            available_props_2 = meshed_region_2.available_property_fields
+
+            # Print the available properties
+            print("Available properties: ", available_props_2)
+
+    .. tab-item:: Fluent
+
+        .. jupyter-execute::
+
+            # Get the available properties
+            available_props_3 = meshed_region_3.available_property_fields
+
+            # Print the available properties
+            print("Available properties: ", available_props_3)
+
+    .. tab-item:: CFX
+
+        .. jupyter-execute::
+
+            # Get the available properties
+            available_props_4 = meshed_region_4.available_property_fields
+
+            # Print the available properties
+            print("Available properties: ", available_props_4)
+
+You can also chose which property you want to extract.
+
+When extracting the properties you get a |PropertyField| with that information. Their data is mapped to
+the entity they are defined at.
+
+Here, we extract the element types for the mesh elements.
+
+The element type is given as a number. See the |element_types| to find the
+corresponding element name.
+
+.. tab-set::
+
+    .. tab-item:: MAPDL
+
+        .. jupyter-execute::
+
+            # Get the element types on the mesh
+            el_types_1 = meshed_region_1.property_field(property_name="eltype")
```

Comment:
There is a dedicated property to get the element types. I wouldn't use `property_field` with the string